src/main/java/cc/hyperium/Hyperium.java

```diff
@@ -108,7 +107,7 @@
 public class Hyperium {
 
     /**
-     * The hyperium instance
+     * The hyperium instanceRichPres
```

Comment:
```suggestion
     * The hyperium instance
```

Reply:
woops

Reply:
hit apply suggestion

Reply:
i manage to always start typing before the search thing comes up with ctrl + shift + f so this happens a lot hahah, i'll fix it

Reply:
lol
you can just apply my change by hitting apply suggestion